### ISSUE TITLE
FusedLocationProviderClient is interace since com.google.android.gms:play-services-location:21.0.0

### ISSUE THREAD

**Issue by thubalek** [rtchagas/pingplacepicker#96]
`FusedLocationProviderClient` is interface since October 13, 2022 release. See https://developers.google.com/android/guides/releases#october_13_2022

It causes that application crashes `with IncompatibleClassChangeError`

```
Fatal Exception: java.lang.IncompatibleClassChangeError: Found interface k5.a, but class was expected (declaration of 'k5.a' appears in base.apk)
       at com.rtchagas.pingplacepicker.ui.PlacePickerActivity.getDeviceLocation(PlacePickerActivity.kt:294)
       at com.rtchagas.pingplacepicker.ui.PlacePickerActivity.initMap(PlacePickerActivity.kt:481)
       at com.rtchagas.pingplacepicker.ui.PlacePickerActivity.access$initMap(PlacePickerActivity.kt:52)
       at com.rtchagas.pingplacepicker.ui.PlacePickerActivity$checkForPermission$1.onPermissionGranted(PlacePickerActivity.kt:272)
       ```

**Comment by cempo** [comment]
The same issue

**Comment by dzungpv** [comment]
I fixed the issue here https://github.com/rtchagas/pingplacepicker/pull/98

**Comment by rtchagas** [comment]
Fixed on release 3.0
